docs/deploying/sua-service.yaml

```diff
@@ -16,7 +16,7 @@ spec:
     spec:
       containers:
         - name: suaservice
-          image: ghcr.io/softwaredefinedvehicle/sdv-self-update-agent/sua:v0.1.16
+          image: ghcr.io/SoftwareDefinedVehicle/leda-contrib-self-update-agent-fork/self-update-agent:latest
```

Comment:
After the successful merge and build it should be changed to eclipse-leda/leda-contrib-self-update-agent

Reply:
Yes, will do as a small separate commit